src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -107,11 +127,11 @@ const NavHeader = (props) => {
           </>
         ) : null}
       </Animated.View>
-      <Animated.View
+      <View
         onLayout={onDomainLayout}
         style={[styles.domain(isHeaderHide), {bottom: getBottomPostition()}]}>
         <TopicDomainHeader {...props} />
-      </Animated.View>
+      </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onLayout` prop has been moved from the `<Animated.View>` to a `<View>`. This change seems reasonable as it's more appropriate to measure the layout of the static `<View>` rather than the animated one.
```diff
- <Animated.View
+ <View
    onLayout={onDomainLayout}
    style={[styles.domain(isHeaderHide), {bottom: getBottomPostition()}]}>
    <TopicDomainHeader {...props} />
- </Animated.View>
+ </View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->